src/VoronoiDelaunay.jl

```diff
@@ -131,6 +132,8 @@ mutable struct DelaunayTessellation2D{T<:AbstractPoint2D}
         t = new(_trigs, 3, Int64[], 0)
         sizehint!(t._edges_to_check, 1000)
         sizehint!(t, n)
+        ranges=(min_coord,max_coord,min_coord,max_coord)
```

Comment:
Setting `ranges` as in line 135 only sets the variable locally inside the function. Move the line to before line 132 and include `ranges` as an argument to the constructor `new` in line 132 like so:
```
_ranges = (min_coord, max_coord, min_coord, max_coord)
t = new(_trigs, 3, Int64[], 0, _ranges)
```